dimagi/pages/templates/partials/footer/nav_section/_base.html

```diff
@@ -1,11 +1,11 @@
-<h4 class="gap-bottom-base {{ css_class }}">
+<h5 class="gap-bottom-base {{ css_class }}">
```

Comment:
was this change made for semantic purposes or styling? if it was made for styling, consider adding the `h5` class instead of changing the tag, as that change also carries semantic meaning.

Reply:
now i revert to h4, but review  UI. If we use h4 first menu's second word going to next line that's not looking good. So let me know how to manage this.